framework_laptop.c

```diff
@@ -774,7 +774,7 @@ static struct platform_driver framework_driver = {
 		.dev_groups = framework_laptop_groups,
 	},
 	.probe = framework_probe,
-	.remove = framework_remove,
+	.remove = (void *) framework_remove,
```

Comment:
This should not be needed.
If it is indeed needed, control flow integrity checking will trap it at runtime.